iroh-automerge-repo/src/main.rs

```diff
@@ -55,10 +82,72 @@ async fn main() -> anyhow::Result<()> {
 
     let args = Args::parse();
 
-    let endpoint = iroh::Endpoint::builder().discovery_n0().bind().await?;
+    // Pull in the secret key from the environment variable if it exists
+    // This key is the public Node ID used to identify the node in the network
+    // If not provided, a random key will be generated, and a new Node ID will
+    // be assigned each time the app is started
+    let secret_key = match std::env::var("IROH_NODE_SECRET_KEY") {
```

Comment:
We kind-of have an [established](https://github.com/n0-computer/sendme/blob/3db5ea170234650b626315d8d18ed106ea1b5612/src/main.rs#L280) [pattern](https://github.com/n0-computer/dumbpipe/blob/4a839b98ac37e83cf4af890fb7d4eb0d9b995d31/src/main.rs#L267) [of](https://github.com/n0-computer/iroh/blob/e30c788f968265bd9d181e5ca92d02eb61ef3d0d/iroh/examples/transfer.rs#L184) using the `IROH_SECRET` environment variable for this.

```suggestion
    let secret_key = match std::env::var("IROH_SECRET") {
```